awesome_book.js

```diff
@@ -0,0 +1,61 @@
+let Book = [];
+
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const displayBooks = document.querySelector('.display-books');
+const add = document.querySelector('.add-book');
+
+class Books {
+  constructor(Title, author) {
+    this.Title = Title;
+    this.author = author;
+  }
+}
+
+function addBook() {
+  let title = bookTitle.value;
+  let author = bookAuthor.value;
+  const book = new Books(title, author);
+  Book.push(book);
+  const div = document.createElement('div');
+  div.innerHTML = `<h2>${book.Title}</h2> <p>${book.author}</p> <button onclick='$remove(this)' class='remove-book'>remove</button> <hr>`;
+  displayBooks.appendChild(div);
+  author = '';
+  title = '';
+}
+
+add.addEventListener('click', addBook);
+
+/* eslint-disable */
+function remove(e) {
+    /* eslint-enable */
+  displayBooks.innerHTML = '';
+  Book = Book.filter((book) => book.Title !== e.parentElement.childNodes[0].textContent);
+
+  /* eslint-disable */
+    Book.map((book) => {
+        /* eslint-enable */
+    const div = document.createElement('div');
+    div.innerHTML = `<h2>${book.Title}</h2> <p>${book.author}</p> <button onclick='remove(this)' class='remove-book'>remove</button> <hr>`;
+    displayBooks.appendChild(div);
+  });
+}
```

Comment:
- [ ] Good job on the functionalities but there is an issue. Remove book functionality isn't working. Please make sure removing books is working fine and users are able to remove the book from the collections after adding it.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/158216717-21f54935-73c2-4a73-91ec-8adef21bf020.png)